.github/workflows/node.js.yml

```diff
@@ -2,18 +2,23 @@ name: Node.js CI
 
 on:
   push:
-    branch: master
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
   pull_request:
-    branch: master
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
 
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
         node-version: [12.x, 14.x, 16.x]
+        os: [macos-latest, ubuntu-latest, windows-latest]
```

Comment:
I ask myself, is this really necessary? :) We're not compiling anything, if node works on the platform, Koa will too. Just seems like more work for no real benefit? Do you have a different opinion here or am I missing something maybe?

Reply:
Nodejs has some APIs that return inconsistent results between win and other platforms(such as path). I think add these platforms can prevent some cross-platform bugs from happening.